fdbserver/workloads/BlobRestoreWorkload.actor.cpp

```diff
@@ -106,40 +107,92 @@ struct BlobRestoreWorkload : TestWorkload {
 					return Void();
 				}
 			}
-			wait(delay(1));
+			wait(delay(5));
 		}
 	}
 
-	ACTOR static Future<Void> copyToOriginalDb(Database cx, BlobRestoreWorkload* self) {
-		state RangeResult data;
-
-		// Read data from restored db
-		state Transaction tr1(self->extraDb_->clone());
+	ACTOR static Future<Standalone<VectorRef<KeyValueRef>>> read(Database cx, BlobRestoreWorkload* self) {
```

Comment:
This probably works fine for small data sizes in simulation, but you could do more of a streaming compare to save memory instead of loading the whole user keyspace into memory from both databases. If this isn't OOMing in simulation then just add it as a TODO I guess.

Reply:
that's a good point. we should not read all rows into memory. updated to read and compare in batches.